pypeit/scripts/coadd_1dspec.py

```diff
@@ -25,16 +28,18 @@ def main(args, unit_test=False, path=''):
     import glob
     import yaml
 
+    import numpy as np
     from numpy import isnan
-    import pdb as debugger
 
     from astropy.io import fits
 
     from pypeit import msgs
-    from pypeit.core import coadd
+    from pypeit.core import coadd1d
     from pypeit import specobjs
     from pypeit.spectrographs import util
 
+    from IPython import embed
```

Comment:
Are we doing `import IPython` or this?
I prefer the latter but let's be consistent
so that I don't have to scan up to the top of
the file everytime I want to add an `embed`

Reply:
Using import IPython, changed.